File: Haskell/469A.hs
import Control.Monad
import Data.List

main :: IO ()
main = getLine >>= (\n ->
    putStrLn . (\m -> if n == m  then "I become the guy." else "Oh, my keyboard!") . length . nub . concatMap (map (read :: String -> Int) . tail . words) =<< replicateM 2 getLine ) . read

File: CPP/469A.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main ()
{
  size_t n, p, q;
  cin >> n;

  vector<size_t> v(n, 1);

  for(cin >> q; q-- > 0; v[p-1] = 0)
    cin >> p;

  for(cin >> q; q-- > 0; v[p-1] = 0)
    cin >> p;

  const auto it = find(v.begin(), v.end(), 1);
  if (it == v.end())
    cout << "I become the guy.";
  else 
    cout << "Oh, my keyboard!";
}